power.mss

```diff
@@ -33,22 +34,44 @@
 }
 
 #power-towers {
-  [power = 'tower'] {
```

Comment:
You still need to keep `[power = 'tower'] {` here so that `power=pole` does not render like power=tower from z14 to z15.

Reply:
Fixed

Reply:
[power = `tower`] not [power = `pole`] :-)